.gitignore

```diff
@@ -8,4 +8,4 @@ Gemfile.lock
 
 # docs
 /.yardoc
-/doc
```

Comment:
:warning: such whitespace-line changes may not be approved when proposing as PR to rouge.